warmup.krun

```diff
@@ -124,16 +123,16 @@ SKIP = [
     #
     #"*:PyPy:*",
     "*:CPython:*",
-    #"*:Hotspot:*",
+    #"*:HotSpot:*",
     #"*:Graal:*",
     #"*:LuaJIT:*",
     #"*:HHVM:*",
-    #"*:JRubyTruffle:*",
+    #"*:TruffleRuby:*",
     #"*:V8:*",
     #"*:C:*",
 
     # This benchmark is slow *and* crashes
-    "fasta:JRubyTruffle:default-ruby",
+    "fasta:TruffleRuby:default-ruby",
```

Comment:
nbody/hhvm has a performance regression. Before it took 4 secs per iter, now 10. This means this benchmark/vm pair alone will take a week. Will disable it.